src/components/common/navBar/NavBar.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+import { NavLink, Link } from 'react-router-dom';
+import SearchBar from '../searchbar';
+import { useOktaAuth } from '@okta/okta-react';
+
+export function NavBar({ searchVisible, data, setData }) {
+  const { authState, authService } = useOktaAuth();
+
+  return (
+    <StyledNavBar className="nav-container">
+      <div className="nav">
+        <div className="logo">
+          <NavLink to="/" activeStyle={{ color: 'black' }}>
+            <span style={{ color: 'rebeccapurple' }}>MERCHANT</span> MARKETPLACE
+          </NavLink>
+        </div>
+        <Link to="/myprofile/inventory">Inventory</Link>
+        <Link>Orders</Link>
+        <Link>Payment</Link>
+        <Link>Messages</Link>
+      </div>
+      <SearchBar searchVisible={searchVisible} setData={setData} data={data} />
+    </StyledNavBar>
+  );
+}
+
+const StyledNavBar = styled.nav`
```

Comment:
Like we agreed, I prefer this method of declaring our styles, it's easier to keep track of the relationship between classes and dom elements.